Adding trace search instead of trace list (#6162)
DataDog/documentation: content/en/tracing/visualization/resource.md

|-------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|
| `Avg Span/trace`  | Average number of occurrences of the span, for traces including the current resource, where the span is present at least once.                     |
| `% of traces`     | Percentage of traces including the current resource where the span is present at least once.                                                       |
| `Avg duration`    | Average duration of the span, for traces including the current resource, where the span is present at least once.                                  |
| `Avg % Exec Time` | Average ratio of execution time for which the span was active, for traces including the current resource, where the span is present at least once. |

**Note**: A span is considered active when it's not waiting for a child span to complete. The active spans at a given time, for a given trace, are all the leaf spans (i.e.: spans without children).

## Traces

> Consult the list of [traces][6] associated with this resource thanks to the [Trace search][7] modal already filtered on your environment, service, operation, and resource name:

Reviewer: ```suggestion
Consult the list of [traces][6] associated with this resource in the [Trace search][7] modal already filtered on your environment, service, operation, and resource name:
```
